R/util.R

```diff
@@ -523,20 +519,6 @@ copy_directory <- function(src, as, rollback_on_error = FALSE) {
   }
 }
 
-expand_directory_list <- function(files) {
```

Comment:
there's a subtlety here with the path before and after the source root, so this logic is now in the function where it's used, though it might come back out here once we get global directories working